test/e2e/preempt.go

```diff
@@ -260,4 +262,60 @@ var _ = Describe("Job E2E Test", func() {
 		err = waitTasksReady(ctx, queue2Job, int(rep)/2)
 		Expect(err).NotTo(HaveOccurred())
 	})
+
+	It("multi-preemptor-jobs who are in different priority", func() {
+		ctx := initTestContext(options{
+			queues: []string{"q1-preemption"},
+			priorityClasses: map[string]int32{
+				highPriority:   highPriorityValue,
+				middlePriority: middlePriorityValue,
+				lowPriority:    lowPriorityValue,
+			},
+		})
+		defer cleanupTestContext(ctx)
+
+		slot := oneCPU
+		rep := clusterSize(ctx, slot)
+		job := &jobSpec{
+			tasks: []taskSpec{
+				{
+					img: defaultNginxImage,
+					req: slot,
+					min: 1,
+					rep: rep,
+				},
+			},
+		}
+
+		job.name = "j1-q1"
+		job.pri = lowPriority
+		job.queue = "q1-preemption"
+		queue1Job := createJob(ctx, job)
```

Comment:
I guess you mean low priority job

Reply:
new PR has optimize code style to make case more readable